twilio/endpoint.bal

```diff
@@ -92,7 +92,7 @@ public client class Client {
     #
     # + messageSid - Message-sid of a relevant message
     # + return - If success, returns a message resource response record, else returns error
-    @display {label: "Get Message"}
+    @display {label: "Get message details"}
```

Comment:
Shall we capitalize first letter of each word in the label, which is the common convention we follow now.